balancer-js/src/modules/pools/apr/apr.ts

```diff
@@ -361,17 +360,16 @@ export class PoolApr {
     );
   }
 
-  private async totalLiquidity(): Promise<string> {
+  private async totalLiquidity(pool: Pool): Promise<string> {
     const liquidityService = new Liquidity(this.pools, this.tokenPrices);
-    const liquidity = await liquidityService.getLiquidity(this.pool);
+    const liquidity = await liquidityService.getLiquidity(pool);
 
     return liquidity;
   }
 
-  private async bptPrice() {
+  private async bptPrice(pool: Pool) {
```

Comment:
I would add a comment here or maybe even rename the function to make it clear that this is an approximate BPT price based on Coingecko/external token prices.

Reply:
good point. added descriptions to functions